docusaurus/sidebars.js

```diff
@@ -174,6 +174,11 @@ const sidebars = {
     collapsible: true,
     collapsed: true,
     items: [
+      {
+        type: 'doc',
+        label: 'Reference',
+        id: 'javascript_typescript/reference',
```

Comment:
Nitpick: I'd go with `javascript-typescript/reference`. Both because of personal preferences, but could be slightly better SEO-wise.

Reply:
If you do the change, remember to change in `_plugin/docusaurus.rb` too

Reply:
Ok. Can you take care of that?  Feel free to merge this PR if you think it's ok. Thanks.

Reply:
nvm, I can do it myself. I did not know if we can merge it but I saw your slack message that we can merge.